browser/src/map/Clipboard.js

```diff
@@ -1170,7 +1170,7 @@ L.Clipboard = L.Class.extend({
 		if (!this._downloadProgress || this._downloadProgress.isClosed())
 			return;
 
-		if (this._downloadProgressStatus() === 'downloadButton')
+		if (['downloadButton', 'confirmPasteButton'].includes(this._downloadProgressStatus()))
```

Comment:
Same here: OK to hide the download widget in more cases, but could you describe why is this necessary? What is the case when we get here with a downloadButton case and when do we get confirmPasteButton?

Reply:
'downloadButton' doesn't seem to be used, added to the commit message.